yui/src/rooms/js/rooms.js

```diff
@@ -75,9 +75,7 @@ M.mod_bigbluebuttonbn.rooms = {
         if (btn) {
             btn.onclick = function () {
                 var copyText = document.getElementById("id_guestlinkurl");
-                copyText.select();
-                copyText.setSelectionRange(0, 99999); /*For mobile devices*/
-                document.execCommand("copy");
+                navigator.clipboard.writeText(copyText.textContent);
```

Comment:
execCommand is deprecated in favour of using the clipboard api directly:

https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand
https://developer.mozilla.org/en-US/docs/Web/API/Clipboard/writeText

Opted to use writeText as it has better browser support (and the write method wouldn't work in firefox)